gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
> also what if you tests code in a deeper folder, and you need a lot of ../, and you decide to restructure your tests code?

`spec/spec_helper.rb` doesn't change locations per standard rspec conventions, the rest of the test code can be restructured without impacting this path.

the only scenario where I see this breaking is if we rename the gem.